go/libkb/assertion.go

```diff
@@ -522,7 +522,7 @@ func (a AssertionSocial) ToLookup() (key, value string, err error) {
 }
 
 func (a AssertionPhoneNumber) ToLookup() (key, value string, err error) {
-	return "phone", a.Value, nil
+	return "phone", "+" + a.Value, nil
```

Comment:
We probably want to do as much canonicalization as possible to phone numbers to avoid unnecessary weird conflicts. Not sure where or if this is already done. Wouldn't want different impteams with "1083475713" and "108 347 5713" and "108-347-5713".

Reply:
It's done on the server side right now:
https://github.com/keybase/keybase/blob/master/lib/checkers.iced#L218-L226

Reply:
Cool. Also the client should get the canonicalized version into impteam invitation sigs